### DISCUSSION TITLE
Convert Utility's Detonate x/y/z coordinates into PlaceName

### DISCUSSION THREAD

**Discussion by dbalders** [saul/demofile#332] [Questions] [answered]
Hello, Is it possible to convert the x y and z values from when a flashbang or other grenades detonate into the PlaceName where it detonated? My thought was to check if it was inside the PlaceName bounding boxes, but, I am not sure those are listed anywhere as I can't find them. May have to manually go into the game and record the bounding areas, but, wanted to see if anyone had a different idea that may be in the demos or csgo files that could be more precise. Thanks.

**Comment by saul** [comment, marked as answer]
It is possible, but not with this library. There are libraries available that can parse CS:GO nav meshes, e.g. https://github.com/mrazza/gonav

**Comment by dbalders** [comment]
Thank you Saul, this is exactly what I was looking for! I appreciate your help.
